src/Sulu/Component/Content/Repository/Content.php

```diff
@@ -214,6 +228,48 @@ public function getNodeType()
     }
 
     /**
+     * Returns template.
+     *
+     * @return string
+     *
+     * @VirtualProperty
+     * @SerializedName("template")
+     */
+    public function getTemplate()
+    {
+        if ($this->brokenTemplate) {
+            return;
+        }
+
+        return $this->template;
+    }
+
+    /**
+     * Returns original-template.
+     *
+     * @return string
+     *
+     * @VirtualProperty
+     * @SerializedName("originalTemplate")
+     */
+    public function getOriginalTemplate()
+    {
+        return $this->template;
+    }
+
+    /**
+     * Set broken-template flag.
+     *
+     * @return $this
+     */
+    public function setBrokenTemplate()
```

Comment:
Since the other setters in this class also take arguments I would do the same for this one.